packages/url_launcher/url_launcher_ios/example/ios/RunnerUITests/URLLauncherUITests.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Foundation
+import XCTest
+
+class URLLauncherUITests: XCTestCase {
+    override func setUp() {
```

Comment:
Apologies we don't enforce our swift-format preferences in CI yet (it's coming! https://github.com/flutter/flutter/issues/41129), but the Flutter repos use 2 spaces for indentation.